frontend/css/wi-volunteer-management-public.css

```diff
@@ -80,4 +80,33 @@
 
 #wivm-sign-up-form .field-error {
 	border: 2px solid #990000;
+}
+
+/* Widget Styles */
+.widget-volunteer-opp-info {
```

Comment:
I think it's fine to remove this. While we don't want it to be too tall, I think forcing users to scroll may be more frustrating. Definitely a good thought though.